configure

```diff
@@ -1197,8 +1197,10 @@ EOT
         open(my $fh, '<', $file) or die "ERROR: Could not open $file: $!";
         while (my $row = <$fh>) {
           chomp $row;
-          unless ($row =~ /^include/) {
-            my ($key, $value) = split(/=/, $row);
+          next if ($row =~ /^\s*#/);
```

Comment:
This line appears to be unnecessary since the comments are already removed by the next line, I don't think we're gaining much by determining that it's a comment-only line.

Reply:
Agreed.